### DISCUSSION TITLE
[预览]v2.0.6

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#2607] [分享 / Share]
- 修复 `隐藏首页横幅` 在 "在线列表" 页面不生效. (#2602)
- 修复 `控制栏触摸优化` 在番剧区失效.
- 更新 `夜间模式`. (#2597)
- `下载视频` 支持杜比视界清晰度. (#2596)

刚发布那会发现编译配置有点小问题, 请确保使用版本大于等于 `v2.0.6-preview-2-ga13ef02f2`, 如果不是请直接用最新的文件覆盖安装.

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.0.6-preview'>[预览]v2.0.6</a>.</em>

**Comment by SekiBetu** [comment]
首页和视频播放页的夜间模式没修复，反而失效了

**Reply by SekiBetu** [reply]
v2.0.6-preview-2-ga13ef02f2
这个版本可以了

**Reply by the1812** [reply]
再更一下夜间模式组件, 详情里 Commit Hash 应该是 `a13ef02f2`

**Reply by selfdialog** [reply]
插件合集在哪儿来着，找了半天没找到

**Reply by rick-yao** [reply]
点击检查更新显示已更新，刷新后生效，但是实际上夜间模式组件还是Commit: cf8b48355，脚本本体是v2.0.6-preview-2-ga13ef02f2
，是我更新的姿势不对么

**Reply by the1812** [reply]
@rick-yao 你通用设置里更新源是哪个

**Reply by rick-yao** [reply]
github和jsdelivr换着来都不行

**Reply by SekiBetu** [reply]
> 点击检查更新显示已更新，刷新后生效，但是实际上夜间模式组件还是Commit: [cf8b483](https://github.com/the1812/Bilibili-Evolved/commit/cf8b483555a7c0450a216cdcdc325d577f4790e6)，脚本本体是v2.0.6-preview-2-ga13ef02f2 ，是我更新的姿势不对么

导出配置文件看一眼更新链接

**Reply by the1812** [reply]
直接拿直链装试试? 
https://raw.githubusercontent.com/the1812/Bilibili-Evolved/preview/registry/dist/components/style/dark-mode.js

**Reply by rick-yao** [reply]
![image](https://user-images.githubusercontent.com/59443488/145206090-458b42d7-a820-4eea-8dc2-38607c359bf0.png)
没找到。。

**Reply by rick-yao** [reply]
> 直接拿直链装试试? https://raw.githubusercontent.com/the1812/Bilibili-Evolved/preview/registry/dist/components/style/dark-mode.js

多谢，用直链就好了

**Reply by SekiBetu** [reply]
> ![image](https://user-images.githubusercontent.com/59443488/145206090-458b42d7-a820-4eea-8dc2-38607c359bf0.png) 没找到。。

找更新链接挺简单的，比如说你拖动右边的滚动条拉到最下面。。。

**Reply by AccLasArika** [reply]
我的夜间模式组件是Commit: cf8b48355，也是无效的，夜间模式字体白色，但底板也是白色，是不是叔叔刚加了一个奇怪的容器啊；我超宽屏两边暗中间白底白字眼睛要瞎了2333

**Reply by AccLasArika** [reply]
哦我也用直链更新解决了，我是傻子对不起

**Reply by rick-yao** [reply]
![image](https://user-images.githubusercontent.com/59443488/145207458-28489886-2279-4356-a07b-2486833ea448.png)
即使更新选择github，url也都是jsdelivr的，除了我刚才手动直链更新的darkmode @the1812 这是正常的么

**Reply by the1812** [reply]
这个 url 表示的是你安装的那个时刻选择的更新源, 还不支持实时切换

**Reply by HAOZ666** [reply]
> 哦我也用直链更新解决了，我是傻子对不起

你好 怎么操作？

**Reply by AccLasArika** [reply]
侧边栏——组件——粘贴url，用https://raw.githubusercontent.com/the1812/Bilibili-Evolved/preview/registry/dist/components/style/dark-mode.js

**Reply by HAOZ666** [reply]
> 侧边栏——组件——粘贴url，用https://raw.githubusercontent.com/the1812/Bilibili-Evolved/preview/registry/dist/components/style/dark-mode.js

thks

**Comment by ghost** [comment]
夜间模式BUG就是今天突然又弄出个更像山寨youtube的新UI的预兆吧，能不能麻烦出一个更改字体大小功能，叔叔是不是以为人均4K屏啊字弄这么大

**Comment by the1812** [comment]
单独发一下, 夜间模式没收到更新的, 可以先直接用直链安装:
https://raw.githubusercontent.com/the1812/Bilibili-Evolved/preview/registry/dist/components/style/dark-mode.js

**Comment by Malus-Mill** [comment]
那个、V1的夜间模式还会修吗？

**Reply by SekiBetu** [reply]
下次更新就没有v1了，v1变v2

**Reply by SekiBetu** [reply]
建议到时候发布的时候让用户重新删掉安装一遍，省的配置文件出现各种奇葩的无意义的问题

**Comment by FoundTheWOUT** [comment]
草，这发版速度，不要过劳而便当了（ 🤣

**Reply by the1812** [reply]
![sticker](https://user-images.githubusercontent.com/26504152/145338192-7dfa7caa-af5f-425b-a13d-893e5b09cdc9.jpg)

**Comment by Blossomers** [comment]
昂大佬能不能请教一下 这个黑夜模式失效该如何操作能修复……小白表示不是很明白用直链安装什么的是啥意思呜呜

**Reply by Geeyun-JY3** [reply]
认真看 README：https://github.com/the1812/Bilibili-Evolved/tree/preview#设置

**Comment by Blossomers** [comment]
感谢感谢…我这就看




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "the1812/Bilibili-Evolved"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年12月10日(星期五) 上午9:11
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [the1812/Bilibili-Evolved] [预览]v2.0.6 (Discussion #2607)





 
认真看 README：https://github.com/the1812/Bilibili-Evolved/tree/preview#设置
 
—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by Blossomers** [comment]
感谢感谢…我这就看
谢谢！已解决问题！


------------------ 原始邮件 ------------------
发件人:                                                                                                                        "the1812/Bilibili-Evolved"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年12月10日(星期五) 上午9:11
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [the1812/Bilibili-Evolved] [预览]v2.0.6 (Discussion #2607)





 
认真看 README：https://github.com/the1812/Bilibili-Evolved/tree/preview#设置
 
—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by kethoven** [comment]
可以修改一些v1版本的吗？

**Reply by SekiBetu** [reply]
马上v1就没了，建议等一段时间直接上v2
